tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -398,7 +412,11 @@ object QualOutputWriter {
       ESTIMATED_GPU_DURATION -> ESTIMATED_GPU_DURATION.size,
       ESTIMATED_GPU_SPEEDUP -> ESTIMATED_GPU_SPEEDUP.size,
       ESTIMATED_GPU_TIMESAVED -> ESTIMATED_GPU_TIMESAVED.size,
-      SPEEDUP_BUCKET_STR -> SPEEDUP_BUCKET_STR_SIZE
+      SPEEDUP_BUCKET_STR -> SPEEDUP_BUCKET_STR_SIZE,
+      UNSUPPORTED_EXECS -> getMaxSizeForHeader(unSupportedExecsSum.map(_ + unSupportedExecsSize),
+        UNSUPPORTED_EXECS),
+      UNSUPPORTED_EXPRS -> getMaxSizeForHeader(unSupportedExprsSum.map(_ + unSupportedExprsSize),
```

Comment:
why are we adding header size to express string size, should just be the longer of the two?

Reply:
Since we are adding delimiter ";" after an exec or expression, we need the count of execs/expressions which is calculated in unSupportedExprsSize so that we get the correct size to print.